examples/FsCheck.XUnit.CSharpExamples/ReverseFixture.cs

```diff
@@ -6,17 +6,25 @@ namespace FsCheck.XUnit.CSharpExamples
 {
     public class ReverseFixture
     {
-        //[Property]
-        //public void Bcl2()
-        //{
-        //}
-
-        [Property]
+        [Property(QuietOnSuccess = true)]
         public bool Bcl(int[] xs)
         {
           return xs.Reverse().Reverse().SequenceEqual(xs);
         }
 
+        [Property(QuietOnSuccess = true)]
+        public void Bcl2(int[] xs)
```

Comment:
Other than helping in debugging, it doesn't seem to provide any sort of *automated* feedback, unless I miss something.